relays/substrate/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "substrate-bridge"
+version = "0.1.0"
+authors = ["Jim Posen <jim.posen@gmail.com>"]
```

Can you update to `Parity Technologies <admin@parity.io>`?
```suggestion
authors = ["Parity Technologies <admin@parity.io>"]
```